maven-surefire-common/src/main/java/org/apache/maven/plugin/surefire/report/Utf8RecodingDeferredFileOutputStream.java

```diff
@@ -42,7 +42,7 @@
     private boolean closed = false;
 
     @SuppressWarnings( "checkstyle:magicnumber" )
-    public Utf8RecodingDeferredFileOutputStream( String channel )
+    Utf8RecodingDeferredFileOutputStream( String channel )
```

Comment:
why removing public constructor this is not backward compat???? if checkstyle complains about this so remove checkstyle usage as it's useless....

Reply:
This constructor wasn't used outside the package. Intellij also confirms it, so checkstyle reports it properly. Also keep in mind that tests passes. If it not being used, access can be safly lowered to package level.